src/models/commentModel.js

```diff
@@ -1,14 +1,29 @@
 import mongoose from 'mongoose';
+const { Schema } = mongoose;
 
 const CommentSchema = new mongoose.Schema({
-  id: {
+  content: {
     type: String,
-    required: true,
-    unique: true
+    required: true
   },
-  commenterId: String,
-  content: String,
-  date: Date
+  commenterName: {
+    type: String,
+    required: true
+  },
+  authorID: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+    required: true
+  },
+  threadID: {
+    type: Schema.Types.ObjectId,
+    ref: 'Thread',
+    required: true
+  },
+  date: {
+    type: Date,
+    default: Date.now
```

Comment:
Mongoose has a built-in createdAt property, please check the documentation.

Reply:
I conducted several comment features from the Internet. And they all use the Date. now. I wonder  if there is any specific reason for me to change to createdAt?

Reply:
Yeah, it is definitely ok to keep it tho.